docs/api-reference/data-fetching/get-initial-props.md

```diff
@@ -68,6 +68,8 @@ For the initial page load, `getInitialProps` will run on the server only. `getIn
 - `getInitialProps` can **not** be used in children components, only in the default export of every page
 - If you are using server-side only modules inside `getInitialProps`, make sure to [import them properly](https://arunoda.me/blog/ssr-and-server-only-modules), otherwise it'll slow down your app
 
+> Note that irrespective of rendering type, any `props` will be passed to the page component and can be viewed on the client-side in the initial HTML. This is to allow the page to be [hydrated](https://reactjs.org/docs/react-dom.html#hydrate) correctly. Make sure that you don't pass any sensitive information that shouldn't be available on the client in `props`.
```

Comment:
Let's keep the format consistent with other notes. **Note:** 

I think we can drop "that irrespective of rendering type"

*Note:** Any `props` returned in `getInitialProps` will be passed to the...

"Make sure that you don't pass any" to just "Don't pass any..."